### PR TITLE
Update Best Practices and Contribution Guide for LFDT

### PR DESCRIPTION
**Pull request by denyeart**
Update Best Practices and Contribution Guide for LFDT.

### CODE REVIEW

File: guidelines/project-best-practices.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 * Document Long-term support (LTS) release strategy - [Fabric example](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0005-lts-release-strategy.md)
```
Comment (denyeart) [inline guidelines/project-best-practices.md]: Do we know which projects will remain in github.com/hyperledger versus move to github.com/LF-Decentralized-Trust?

File: guidelines/project-best-practices.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 
 * Foster a welcoming, positive, and public environment where contributions are encouraged. - See [Creating an Inclusive Community presentation](https://www.youtube.com/watch?v=lHOINFlb9JA)
 * Decisions should be made in public, or at least socialized in public
-* Engage on Discord Chat - see [Suggested Channel Organization and Naming](https://wiki.hyperledger.org/display/TF/Hyperledger%27s+Transition+to+Discord+Proposal)
+* Engage on Discord Chat - see [Suggested Channel Organization and Naming](https://wiki.hyperledger.org/display/TF/Transition+to+Discord+Proposal)
```
Comment (denyeart) [inline guidelines/project-best-practices.md:75]: Do we know what the new wiki address will be?

File: guidelines/project-best-practices.md [hunk 3]
```diff
@@ -88,10 +88,10 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 * Present at Meetups - Virtual or in person – these are well attended and the videos also get many views
   - Email meetup@hyperledger.org if you're interested in presenting or join one of the [bi-weekly Meetup and Workshop planning calls every other Thursday at 9:30 AM pacific](https://lists.hyperledger.org/g/meetups/calendar)
```
Comment (denyeart) [inline guidelines/project-best-practices.md]: Do we know what the mail list server address will be?

**Review by denyeart — COMMENTED** [review]
Do we know what will replace github.com/hyperledger-labs?

Answer - will keep hyperledger-labs for now.

File: guidelines/project-best-practices.md (hunk 3 above)
Comment (ryjones) [reply guidelines/project-best-practices.md]: https://lists.lfdecentralizedtrust.org/g/main

File: guidelines/project-best-practices.md (hunk 1 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: Only Besu moving, no changes needed here...

File: guidelines/project-best-practices.md (hunk 2 above)
Comment (denyeart) [reply guidelines/project-best-practices.md:75]: old wiki will remain, no changes needed here.

File: guidelines/project-best-practices.md (hunk 3 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: updated.

**Review by petermetz — APPROVED** [review]

File: guidelines/project-best-practices.md (hunk 2 above)
Comment (tkuhrt) [inline guidelines/project-best-practices.md:75]: I am guessing this wiki page will not be renamed.

File: guidelines/project-best-practices.md [hunk 4]
```diff
@@ -112,18 +112,18 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 ## Security
 
 * Provide three named security contacts per project
-* Define security issue reporting process in SECURITY.md with reference to [Hyperledger reporting process](https://wiki.hyperledger.org/display/SEC/Defect+Response)
+* Define security issue reporting process in SECURITY.md with reference to [Security reporting process](https://wiki.hyperledger.org/display/SEC/Defect+Response)
```
Comment (tkuhrt) [inline guidelines/project-best-practices.md]: Shouldn't this point to the new Security policy that we developed within the TOC (soon to be TAC website)?

File: guidelines/project-best-practices.md [hunk 5]
```diff
@@ -112,18 +112,18 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
...
+* Engage with LFDT staff on possibility of [security audits](https://wiki.hyperledger.org/display/SEC/Security+Code+Audits) for Graduated project major releases; address audit results and socialize
 * Review [OpenSSF secure developer guide](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Concise-Guide-for-Developing-More-Secure-Software.md) and OpenSSF overview presentation to TAC ([charts](https://wiki.hyperledger.org/download/attachments/80775801/OpenSSF%20Hyperledger%2020230119.pdf?version=1&modificationDate=1674218724000&api=v2), [replay](https://youtu.be/0AFFIDKFNvc))
 * Review and obtain OpenSSF Best Practices Badge - see [criteria](https://bestpractices.coreinfrastructure.org/en/criteria/0)
-* Sign release artifacts (TBD) - see proposed [Security Artifact Signing task force](https://github.com/hyperledger/toc/issues/49)
+* Sign release artifacts (TBD) - see proposed [Security Artifact Signing task force](https://github.com/LF-Decentralized-Trust/tac/issues/49)
```
Comment (tkuhrt) [inline guidelines/project-best-practices.md]: I don't think we will be moving the issues over to LF-Decentralized-Trust/tac. Probably should continue to reference hyperledger/toc repo.

File: guidelines/project-best-practices.md [hunk 6]
```diff
@@ -143,7 +143,7 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 * Document Long-term support (LTS) release strategy - [Fabric example](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0005-lts-release-strategy.md)
 * Use Github Actions to automate release process, e.g. publish artifacts and release notes upon drafting a GitHub release
 * Use [Reproducible builds](https://reproducible-builds.org/)
-* Sign release commits with [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (-S or --gpg-sign); sign release artifacts - see [Security Artifact Signing task force](https://github.com/hyperledger/toc/issues/49)
+* Sign release commits with [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (-S or --gpg-sign); sign release artifacts - see [Security Artifact Signing task force](https://github.com/LF-Decentralized-Trust/tac/issues/49)
```
Comment (tkuhrt) [inline guidelines/project-best-practices.md]: Same comment here.

File: guidelines/project-best-practices.md [hunk 7]
```diff
@@ -165,7 +165,7 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
   - Note - just because a change passes automated checks doesn't mean it is necessarily good, it still requires judicious maintainer review
 * Test coverage reporting - run on-demand or nightly
 * Keep CI clean and green at all times, address failures and flakes
-* See also proposed [Automated Pipelines task force](https://github.com/hyperledger/toc/issues/44)
+* See also proposed [Automated Pipelines task force](https://github.com/LF-Decentralized-Trust/tac/issues/44)
```
Comment (tkuhrt) [inline guidelines/project-best-practices.md]: Same comment here.

File: guidelines/project-best-practices.md (hunk 2 above)
Comment (denyeart) [reply guidelines/project-best-practices.md:75]: I already renamed this wiki page to the new link without 'Hyperledger'  :-)

File: guidelines/project-best-practices.md (hunk 4 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: done

File: guidelines/project-best-practices.md (hunk 5 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: done

File: guidelines/project-best-practices.md (hunk 7 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: done

File: guidelines/project-best-practices.md (hunk 6 above)
Comment (denyeart) [reply guidelines/project-best-practices.md]: done

**Comment by tkuhrt** [comment]
See https://github.com/LF-Decentralized-Trust/governance/pull/34 for the Github Contribution Guide

**Comment by tkuhrt** [comment]
See https://github.com/LF-Decentralized-Trust/governance/pull/36 for project best practices.

**Pull request closed without merging**